src/current/v24.1/molt-fetch.md

```diff
@@ -599,14 +647,16 @@ The following `molt fetch` command uses `COPY FROM` to load all tables directly
 {% include_cached copy-clipboard.html %}
 ~~~ shell
 molt fetch \
---source 'postgres://root@localhost:26257/defaultdb?sslmode=verify-full' \
---target 'postgres://root@localhost:26258/defaultdb?sslmode=verify-full' \
+--source 'postgres://root@localhost:26257/defaultdb?sslmode=disable' \
```

Comment:
It may be helpful to emphasize again this is only if the user cannot connect securely to their database, or for testing purposes.